src/database/schema/UserSchema.ts

```diff
@@ -347,15 +388,15 @@ class UserSchema implements BaseDbSchema {
     return new UserSchema({
       id: uuidv4(),
       channelId,
-      userId: object?.user_id,
+      userId: object?.user?.id || object?.user_id,
       bio: object?.bio,
       countryCode: object?.country_code,
-      createdAt: object?.created_at,
-      lastActiveAt: object?.last_active_at,
-      profilePicture: object?.profile_pic_path,
+      createdAt: object?.user?.created_at,
+      lastActiveAt: object?.last_active_at || new Date().toISOString(),
+      profilePicture: object?.user?.image || DEFAULT_PROFILE_PIC_PATH,
       updatedAt: object?.updated_at,
-      username: object?.username,
-      isBanned: object?.is_banned,
+      username: object?.user?.username,
+      isBanned: object?.user?.banned,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There seems to be a change in how the `userId` and `isBanned` fields are being populated. Ensure that this change is intentional and that it doesn't break any existing functionality that relies on these fields. Also, consider adding error handling here similar to what was done in the `fromMemberWebsocketObject` method.


<!-- This is an auto-generated comment by OSS CodeRabbit -->